common/styles/reset.css

```diff
@@ -16,6 +16,8 @@
 html {
   box-sizing: border-box;
   text-size-adjust: 100%; /* 1 */
+  -webkit-font-smoothing: antialiased;
```

Comment:
The linter is complaining about the order of these. I think they should be ordered as if they both said 'font-smoothing'. Like this:

```
box-sizing: border-box;
-moz-osx-font-smoothing: grayscale;
-webkit-font-smoothing: antialiased;
text-size-adjust: 100%; /* 1 */
```

With `-moz` before `-webkit`